parachain/pallets/inbound-queue/Cargo.toml

```diff
@@ -3,16 +3,20 @@ name = "snowbridge-inbound-queue"
 description = "Snowbridge Inbound Queue"
 version = "0.1.1"
 edition = "2021"
-authors = [ "Snowfork <contact@snowfork.com>" ]
+authors = ["Snowfork <contact@snowfork.com>"]
 repository = "https://github.com/Snowfork/snowbridge"
 
 [package.metadata.docs.rs]
-targets = [ "x86_64-unknown-linux-gnu" ]
+targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 serde = { version = "1.0.164", optional = true }
-codec = { version = "3.1.5", package = "parity-scale-codec", default-features = false, features = [ "derive" ] }
-scale-info = { version = "2.7.0", default-features = false, features = [ "derive" ] }
+codec = { version = "3.1.5", package = "parity-scale-codec", default-features = false, features = [
```

Comment:
I prefer when these are on the same line. It makes searching for packages, versions and features easy with grep as the full dependency is on one line.